qase-jest/examples/test/arith.test.js

```diff
@@ -1,5 +1,5 @@
 const { add, mul, sub, div } = require('./arith');
-const { qase } = require('jest-qase-reporter/dist/jest');
+const { qase } = require('jest-qase-reporter/dist/src/jest');
```

Comment:
Why is it changed?

Reply:
Because our code is now built and put into dist/src. The reason for this is the use of `import lockfile from "../package-lock.json"`.
So in this case the typescript compiler creates a package-lock inside the dist folder. TSC puts all compiled code into dist/src to avoid import path problems.